kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java

```diff
@@ -136,4 +157,76 @@ public void testPrometheusProcessAndTaskMetrics() {
                 format("kie_server_task_exited_total{deployment_id=\"prometheus\",process_id=\"per-process-instance-project.usertask\",task_name=\"First task\",} %d.0", exitedTasks)
         );
     }
+
+    @Test
+    public void testPrometheusCaseMetrics() {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertThat(caseId).isNotEmpty();
+
+        assertThat(getMetrics()).contains(
+              "kie_server_case_started_total{case_definition_id=\"" + CASE_DEF_ID + "\",",
+              "kie_server_case_running_total{case_definition_id=\"" + CASE_DEF_ID + "\",} 1.0"
+        );
+
+        caseClient.cancelCaseInstance(CONTAINER_ID_CASE, caseId);
+
+        assertThat(getMetrics()).contains(
+              "kie_server_case_started_total{case_definition_id=\"" + CASE_DEF_ID + "\",",
+              "kie_server_case_running_total{case_definition_id=\"" + CASE_DEF_ID + "\",} 0.0",
+              "kie_server_case_duration_seconds_count{case_definition_id=\"" + CASE_DEF_ID + "\",",
+              "kie_server_case_duration_seconds_sum{case_definition_id=\"" + CASE_DEF_ID + "\","
+        );
+    }
+
+    @Test
+    public void testPrometheusJobMetrics() throws Exception {
+        int currentNumberOfCancelled = jobServicesClient.getRequestsByStatus(Collections.singletonList(STATUS.CANCELLED.toString()), 0, 1000).size();
+        int currentNumberOfDone = jobServicesClient.getRequestsByStatus(Collections.singletonList(STATUS.DONE.toString()), 0, 1000).size();
+
+        Calendar tomorrow = Calendar.getInstance();
```

Comment:
Nitpicking here. Maybe a one-liner will be more clear?

Reply:
Good  point, adjusted.